ql/src/java/org/apache/hadoop/hive/ql/exec/MoveTask.java

```diff
@@ -348,6 +348,17 @@ public TaskInformation(Task task, String path) {
 
   @Override
   public int execute() {
+    if (work != null && work.getLoadTableWork() != null && work.getLoadTableWork().isUseAppendForLoad()) {
```

Comment:
is this the right place to insert this logic? I can see below log trace for MoveTask start + 'explain analyze' logic check.
Do we have q test for `explain analyze LOAD DATA ...`

Reply:
changed & added test for analyse